src/rpc/methods/eth.rs

```diff
@@ -1299,18 +1290,17 @@ impl RpcMethod<2> for EthGetCode {
             gas_limit: BLOCK_GAS_LIMIT,
             ..Default::default()
         };
-        let mut api_invoc_result = None;
-        for ts in ts.chain_arc(ctx.store()) {
-            match ctx.state_manager.call(&message, Some(ts)) {
-                Ok(res) => {
-                    api_invoc_result = Some(res);
-                    break;
+
+        let api_invoc_result = 'invoc: {
+            for ts in ts.chain_arc(ctx.store()) {
+                match ctx.state_manager.call(&message, Some(ts)) {
+                    Ok(res) => {
+                        break 'invoc res;
```

Comment:
The loop label here is unnecessary. Non-blocking.

Reply:
Could you elaborate? I can compile the code without the loop label

```
error[E0571]: `break` with value from a `for` loop
    --> src/rpc/methods/eth.rs:1298:25
     |
1295 |             for ts in ts.chain_arc(ctx.store()) {
     |             ----------------------------------- you can't `break` with a value in a `for` loop
...
1298 |                         break res;
     |                         ^^^^^^^^^ can only break with a value inside `loop` or breakable block
     |
help: use `break` on its own without a value inside this `for` loop
     |
1298 |                         break;
     |                         ~~~~~
```